api/src/controllers/peers.js

```diff
@@ -25,8 +26,28 @@ function PeersControllerFactory (auth, config, log, Peer, connector) {
       router.get('/peers/:id/settlement_methods', auth.checkAuth, this.checkAdmin, this.getSettlementMethods)
       router.delete('/peers/:id', auth.checkAuth, this.checkAdmin, this.deleteResource)
 
-      // Public
-      router.post('/peers/rpc', this.rpc)
+      // authenticated by the plugin being connected
+      router.post('/peers/rpc', this.checkPeerAuth, this.rpc)
+    }
+
+    static * checkPeerAuth (next) {
+      const prefix = this.query.prefix
+      const auth = this.request.headers.authorization
+
+      if (typeof prefix !== 'string' || typeof auth !== 'string') {
+        this.status = 401
+        return
+      }
+
+      const [ , authToken ] = auth.match(/^Bearer (.+)$/) || []
```

Comment:
https://tools.ietf.org/html/rfc6750#section-2.1 it looks like it is, as far as I can tell